INSTALL.md

```diff
@@ -42,29 +41,26 @@ This requires a MongoDB server to be running on your desired host machine.
 #### Python setup
 
 The next step is to set up a Python environment that contains all of the required dependencies with the correct versions.
-You will need Python 3.10 or higher to run *datalab*; we recommend using
-something like [`pyenv`](https://github.com/pyenv/pyenv) to manage Python versions on your machine, to avoid breakages based on your OS's Python versioning.
+You will need Python 3.10 or higher to run *datalab*; we recommend using a tool
+to manage Python versions on your machine, to avoid breakages based on your OS's Python versioning (e.g., [`pyenv`](https://github.com/pyenv/pyenv) or [`uv`](https://github.com/astral-sh/uv)).
 
-##### Using virtual environments with `uv` or `venv`
+##### Installation with `uv` or `venv`
 
-We recommend using a virtual environment tool of your choice to manage the dependencies for the
-Python server, for example [`uv`](https://github.com/astral-sh/uv) (see
-repository for installation instructions), or the
-standard library Python `venv` module.
+We recommend using [`uv`](https://github.com/astral-sh/uv) (see the linked repository or https://docs.astral.sh/uv for installation instructions) for managing your *datalab* installation.
+
+You could also use the standard library `venv` module, but this will not allow you to install pinned dependencies as easily, and is significantly slower than `uv`.
 
 1. Create a virtual environment for *datalab*, ideally inside the `pydatalab` directory.
-    - For `uv`, this can be done with `uv venv`.
+    - For `uv`, this will be done automatically on installation.
```

Comment:
```suggestion
    - When installing using `uv sync`, this will be done automatically on installation.
```

Because `uv pip install...` doesn't create the venv